suricata/update/main.py

```diff
@@ -424,14 +424,25 @@ def manage_classification(suriconf, files):
 def handle_dataset_files(rule, dep_files):
     if not rule.enabled:
         return
-    load_attr = [el.strip() for el in rule.dataset.split(",") if "load" in el][0]
-    dataset_fname = os.path.basename(load_attr.split(" ")[1])
+    load_attr = [el.strip() for el in rule.dataset.split(",") if "load" in el]
+    state_attr = [el.strip() for el in rule.dataset.split(",") if "state" in el]
+    if load_attr and state_attr:
+        logger.error("Invalid dataset rule")
+        return
```

Comment:
Oh, but this might error on a `save` which it shouldn't do.  @inashivb We should probably only log an error if its a dataset rule that needs action, like a load.. Otherwise return early.  We don't need to validate all dataset types here.

Reply:
Indeed did not think about `save`. Should add tests for all options to avoid these issues.

Reply:
I just realized the checks here are probably not very strict, what is someone has `load,save.csv`, we'll return early since `save` was found in the string. Making it better..